packages/babel-core/src/errors/rewrite-stack-trace.ts

```diff
@@ -61,11 +61,13 @@ function CallSite(filename: string): CallSite {
     isConstructor: () => false,
     isToplevel: () => true,
     getFileName: () => filename,
-    getLineNumber: () => undefined,
-    getColumnNumber: () => undefined,
-    getFunctionName: () => undefined,
-    getMethodName: () => undefined,
-    getTypeName: () => undefined,
+    // eslint-disable unicorn/no-useless-undefined
```

Comment:
Is this comment needed, now that you removed undefined from the function bodies?

Reply:
Author of `unicorn/no-useless-undefined` here. 

We fix `() => undefined` to `() => {}`

Reply:
Yes, I understand. Actually the problem is that `eslint-disable` must use `/**/`.

Reply:
To be honest, I personally think `() => undefined` is a little clearer than `() => {}`, especially in
```ts
   return Object.create({
     isNative: () => false,
     isConstructor: () => false,
     isToplevel: () => true,
     getFileName: () => filename,
     /* eslint-disable unicorn/no-useless-undefined */
     getLineNumber: () => undefined,
     getColumnNumber: () => undefined,
     getFunctionName: () => undefined,
     getMethodName: () => undefined,
     getTypeName: () => undefined,
     /* eslint-enable unicorn/no-useless-undefined */
     toString: () => filename,
   } as CallSite);
```
`() => {}` requires more thought to distinguish it from `() => ({})`.